### PR TITLE
Codecov badge

### PR DESCRIPTION
**Pull request by lock9**
@shargon fixed.
I got the info here:
https://gitlab.com/gitlab-org/gitlab-ce/issues/55912

But instead of using  "gl" (gitlab), I used "github".

### CODE REVIEW

**Comment by lock9** [comment]
Shargon, you sent me a coveralls url today, I think we are using codecov. Please let me know if this is correct.

**Comment by vncoelho** [comment]
Lets apply lines counter before merging, wait 1 day plz.

**Comment by shargon** [comment]
Could you replicate this across all projects

File: README.md [hunk 1]
```diff
@@ -5,8 +5,9 @@
 </p>
 
 <p align="center">
-  <a href='https://coveralls.io/github/neo-project/neo-vm?branch=master'>
-    <img src='https://coveralls.io/repos/github/neo-project/neo-vm/badge.svg?branch=master' alt='Current Coverage Status.' />
+  <a href='https://codecov.io/github/neo-project/neo-vm/'>
+    <img src='https://codecov.io/github/neo-project/neo-vm/branch/master/graph/badge.svg' 
+    alt='Current Coverage Status.' />
```
Comment (igormcoelho) [inline README.md]: Link changed then.. nice.

**Comment by lock9** [comment]
I will add the line count, please don't merge it yet

**Comment by shargon** [comment]
We should follow the same order in all projects, i think that the frst one should be `build`

**Comment by lock9** [comment]
@shargon  Is this order good for you?
Screen Shot 2019-07-10 at 21 24 47

**Comment by vncoelho** [comment]
Looks beautiful! Nice job, Rick!

**Comment by vncoelho** [comment]
@lock9 S2, can you proceed with this for other repos?

**Comment by lock9** [comment]
@vncoelho I will do that 👍

**Pull request merged**